README.md

```diff
@@ -1,2 +1,77 @@
-# java-dynamic-sqs-listener
-Sets up a SQS Listener that can be dynamically configured while it is executing.
+[![Build Status](https://travis-ci.org/JaidenAshmore/java-dynamic-sqs-listener.png)](https://travis-ci.org/JaidenAshmore/java-dynamic-sqs-listener)
+[![Coverage Status](https://coveralls.io/repos/github/JaidenAshmore/java-dynamic-sqs-listener/badge.svg?branch=master)](https://coveralls.io/github/JaidenAshmore/java-dynamic-sqs-listener?branch=master)
+
+# Java Dynamic SQS Listener - SQS Listener with easy configuration and extensibility
+SQS Listener implementation that is designed from the ground up to allow for easier configuration and extensibility.
+
+## Why Dynamic SQS Listener?
+Whilst the other implementations of SQS listeners were easy to set up and provided most use cases, they don't provide the extensibility and dynamic
```

Comment:
Whilst? What exactly does that mean Morty!?

Reply:
> Both while and whilst are ancient, though while is older. There's no difference in meaning between them. For reasons that aren't clear, whilst has survived in British English but has died out in the US. However, in Britain it is considered to be a more formal and literary word than its counterpart.

Better dumb down the docs for you americans ;)